src/python/grapl-template-generator/grapl_template_generator/grapl-templates/rust-grpc-service/{{cookiecutter.project_slug}}/build.rs

```diff
@@ -0,0 +1,14 @@
+fn main() {
+    let config = tonic_build::configure();
+
+    config
+        .build_server(true)
+        .build_client(true)
+        .compile(
+            &[
+                "{{cookiecutter.proto_path}}{{cookiecutter.snake_project_name}}.proto",
```

Comment:
This seems like it's going back to the old pattern of embedding protobuf definitions within an individual project, rather than in the top-level unified hierarchy in src/proto. What's the rationale for this reversal?